src/kudu/consensus/log_cache.cc

```diff
@@ -75,6 +79,12 @@ METRIC_DEFINE_gauge_int64(server, log_cache_num_ops, "Log Cache Operation Count"
 METRIC_DEFINE_gauge_int64(server, log_cache_size, "Log Cache Memory Usage",
                           MetricUnit::kBytes,
                           "Amount of memory in use for caching the local log.");
+METRIC_DEFINE_gauge_int64(server, log_cache_msg_size, "Log Cache Message Size",
+                          MetricUnit::kBytes,
+                          "Amount of memory of all messages cached. This is "
+                          "the uncompressed size of the memory. If compression"
```

Comment:
nit: Size of the incoming uncompressed payload for the messages in the log_cache